CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Kubermatic 2.20
 
+## [v2.20.2](https://github.com/kubermatic/kubermatic/releases/tag/v2.20.2)
+
+### Misc
+
+- Add support for configuration annotations and loadBalancerSourceRanges for front-loadbalancer service of node port proxy ([#9486](https://github.com/kubermatic/kubermatic/issues/9486))
+- For the Seed CR, `spec.nodeportProxy.annotations` is deprecated and `spec.nodeportProxy.envoy.loadBalancerService.annotations` should be used instead ([#9486](https://github.com/kubermatic/kubermatic/issues/9486))
+- The Image-loader utility now includes all the images used by KKP, considering also the provider-specific ones (e.g., CCM, CSI) ([#9518](https://github.com/kubermatic/kubermatic/issues/9518))
+- Enable the "vsphereCSIClusterID" feature flag when running the CCM/CSI migration ([#9557](https://github.com/kubermatic/kubermatic/issues/9557))
+- For Kubernetes 1.22 and higher, etcd is updated to v3.5.3 to fix data consistency issues as reported by upstream developers ([#9606](https://github.com/kubermatic/kubermatic/issues/9606))
+
+
+
+
```

Comment:
Nit: These empty lines should be removed.

Reply:
Quite the opposite: They should be there. Someone recently stopped my tradition of visually separating the text blocks in this document, _and I don't like it!_ :trollface:

Reply:
I'm not sure why you would need that since the headlines separate the document already, but sure, I don't mind :P